build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -17,20 +17,18 @@ if($Help)
     exit 0
 }
 
-$Architecture='x64'
-
 $RepoRoot = "$PSScriptRoot\..\.."
 $AppPath = "$PSScriptRoot"
 
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows"
```

Comment:
This is why our $(OutputDir) uses the $RID, so we can build for anything we want, and the build artifacts are separated.